tests/zfs-tests/tests/functional/zvol/zvol_common.shlib

```diff
@@ -140,3 +140,11 @@ function set_blk_mq
 		log_must set_tunable32 VOL_USE_BLK_MQ $1
 	fi
 }
+
+# enable/disable zvol sync mode
+#
+# $1: 1 = enable, 0 = disable
+function set_zvol_sync
+{
+	log_must set_tunable32 VOL_REQUEST_SYNC $1
+}
```

Comment:
Make sure you are calling `save_tunable/restore_tunable` in `zpool_stress.ksh`  so your `VOL_REQUEST_SYNC` value changes are localized to the test.

Reply:
The zvol_sync parameter is switched back to default value under cleanup() function.

Reply:
You can't assume `VOL_REQUEST_SYNC` was set to 0 prior to the test, nor can you assume 0 will be the default in future versions of ZFS.  That's why the convention is to use `save_tunable/restore_tunable` so that all changes to module params are localized to your test.

Reply:
Got it. But the VOL_USE_BLK_MQ should be modified by this way too. I've applied save_tunable/restore_tunable to VOL_USE_BLK_MQ too, and it require to modify save_tunable/restore_tunable functions in the libtest.shlib, because the VOL_USE_BLK_MQ parameter is not present of FreeBSD side. Is it right way? Or I should open independent PR for save_tunable/restore_tunable functions modification? Or should not touch VOL_USE_BLK_MQ at all?